test/unit/index.js

```diff
@@ -5,7 +5,8 @@ var chai = require("chai"),
     sinon = require("sinon"),
     mocha = require("mocha");
 
-var Buttercup = require("../../source/module.js");
+var Buttercup = require("../../source/module.js"),
```

Comment:
That being said, like on my other comment, I'm trying not to change everything as I'd like to do an es6 overhaul MR after these guys..